test/org/sagebionetworks/bridge/dynamodb/DynamoUserConsentDaoTest.java

```diff
@@ -60,15 +66,31 @@ public void canConsentToStudy() {
         assertTrue(userConsentDao.hasConsented(HEALTH_CODE, new StudyIdentifierImpl(consent.getStudyKey())));
 
         // Withdraw
-        userConsentDao.withdrawConsent(HEALTH_CODE, STUDY_IDENTIFIER);
+        userConsentDao.withdrawConsent(HEALTH_CODE, STUDY_IDENTIFIER, DateTime.now().getMillis());
         assertFalse(userConsentDao.hasConsented(HEALTH_CODE, new StudyIdentifierImpl(consent.getStudyKey())));
+        
+        // There should be no consent
+        try {
+            userConsentDao.withdrawConsent(HEALTH_CODE, STUDY_IDENTIFIER, DateTime.now().getMillis());
+            fail("Should have thrown exception");
+        } catch(BridgeServiceException e) {
+            assertEquals(404, e.getStatusCode());
+            assertEquals("Consent not found.", e.getMessage());
+        }
+        
+        UserConsent existingConsent = userConsentDao.getActiveUserConsent(HEALTH_CODE, STUDY_IDENTIFIER);
+        assertNull(existingConsent);
 
         // Can give consent again if the previous consent is withdrawn
         userConsentDao.giveConsent(HEALTH_CODE, consent, DateTime.now().getMillis());
         assertTrue(userConsentDao.hasConsented(HEALTH_CODE, new StudyIdentifierImpl(consent.getStudyKey())));
 
+        // Now we can find it
+        existingConsent = userConsentDao.getActiveUserConsent(HEALTH_CODE, STUDY_IDENTIFIER);
+        assertNotNull(existingConsent);
```

Comment:
Added (added call to getActiveConsent() when there is non to verify it returns false